scissors/src/main/java/com/lyft/android/scissors/GlideFillViewportTransformation.java

```diff
@@ -49,11 +55,21 @@ protected Bitmap transform(BitmapPool bitmapPool, Bitmap source, int outWidth, i
     }
 
     @Override
-    public String getId() {
-        return getClass().getName();
+    public boolean equals(Object obj) {
```

Comment:
We really need to implement `equals()` and `hashCode()` according to the [`Transformation` documentation](https://github.com/bumptech/glide/blob/master/library/src/main/java/com/bumptech/glide/load/Transformation.java#L59) (and the use case mentioned before).

Reply:
Is this enough to implement? since the object is different depending on the viewportWidth/viewportHeight fields, shouldn't we consider those?

Reply:
That's a good point. The use case doesn't have any state but we do. I'll update that.